examples/src/main/java/ai/timefold/solver/examples/conferencescheduling/score/ConferenceSchedulingConstraintProvider.java

```diff
@@ -138,7 +138,7 @@ Constraint roomConflict(ConstraintFactory factory) {
     }
 
     Constraint speakerUnavailableTimeslot(ConstraintFactory factory) {
-        return factory.forEachIncludingNullVars(Talk.class)
+        return factory.forEachIncludingUnassigned(Talk.class)
```

Comment:
I think the Talk still uses `nullable=true`, since I don't see it in the changed files

Reply:
`ConferenceScheduling` never used `nullable = true`.
This code is there for construction heuristics, because this problem has 2 genuine variables.